fdbserver/CMakeLists.txt

```diff
@@ -147,7 +148,6 @@ if (WITH_ROCKSDB_EXPERIMENTAL)
   else()
     target_include_directories(fdbserver PRIVATE ${ROCKSDB_INCLUDE_DIR})
     target_link_libraries(fdbserver PRIVATE fdbclient metacluster sqlite ${ROCKSDB_LIBRARIES} ${lz4_STATIC_LIBRARIES})
-    target_compile_definitions(fdbserver PRIVATE)
```

Comment:
what's the behavior difference with and without this line?

Reply:
This line has no behavior, as it adds no definitions to the compiler, so basically a NO-OP